wbia/dbio/ingest_ggr.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
Gotta bump this line up and remove the space.

Reply:
Hmm... turns out this works, so nevermind. 🤯

Reply:
This was resolved (and temp.txt was removed) in c0037fa1aa54cbba35b4335f8c47db2c129c1ef0